schemas/host.yml

```diff
@@ -79,3 +79,14 @@
       example: "x86_64"
       description: >
         Operating system architecture.
+
+    - name: domain
+      level: extended
+      type: keyword
+      short: Name of the directory the group is a member of.
+      description: >
+        Name of the directory the group is a member of.
+
+        For example, an LDAP or Active Directory domain name.
+      example: CONTOSO
```

Comment:
🤔 I expected this would look more like a domain name, actually.

I suppose it's something optional? Companies that run their domain 100% internal could name their domain CONTOSO, and if the company wanted to make this publicly accessible, it may be "contoso.net"?

Or was the `.fqdn` discussion meant to be about the FQDN of the domain controller? When I read it, I understood `host.fqdn` to be the fqdn of the host, which could in this case be "bob-laptop.CONTOSO"?

Reply:
> same domain Microsoft uses in their examples

Love the approach in any case :-)

Reply:
Hi @webmat it is only a name (can be anything) that refers to an Active Directory Domain or Windows Domain.
It is not optional, always exists. If you do not have an AD the domain name in the event is the name of the local computer. 
Users, Groups and Computers (and in general, objects within the domain) are referenced as domain_name\object-name.
In the case of bob's laptop the reference will be CONTOSO\bob.laptop 
It is not related to domain name in the context of DNS nor with the fact that the domain permit the object will be publicly accesible.
Real domain name examples and user names

An old company I worked for (and doesn't exist anymore :) )
domain: NEXTEL_AR

My home local domain
TargetUserName	Anna
TargetDomainName	DESKTOP-4564S0K

Either is an Active Directory or an stand alone computer in a private network , Windows always asigns a domain name :)